### ISSUE TITLE
Unclear What's New instructions

### ISSUE THREAD

**Issue by story645** [matplotlib/matplotlib#13853]
Trying to make the what's new and following the [update and validate the docs](https://matplotlib.org/devdocs/devel/release_guide.html#update-and-validate-the-docs) guide and it says:
```
comment out the next whats new glob at the top
```
but it's already commented out [in what's new](https://raw.githubusercontent.com/matplotlib/matplotlib/master/doc/users/whats_new.rst)
```
.. _whats-new:
```
Also the instructions in the what new:
```
   For a release, add a new section after this, then comment out the include
   and toctree below by indenting them. Uncomment them after the release.

  .. include:: next_whats_new/README.rst
```
kind of conflict with these instructions from the guide:
```
copy the current content to a file in doc/users/prev_whats_new
merge all of the files in doc/users/next_whats_new/ into doc/users/whats_new.rst and delete the individual files
```
If you do both, you end up with duplicated entries.  

Also, possibly a separate PR/discussion, but what about having some set of categories for what's new sections?

**Comment by dstansby** [comment]
This looks fixed now in master?

**Comment by tacaswell** [comment]
I think this is fixed on both master and 3.1 now that we use `ifconfig`.
